### DISCUSSION TITLE
Autogrow in QInput API does not work on Firefox (Android and Windows)

### DISCUSSION THREAD

**Discussion by MandingaRadi** [quasarframework/quasar#15195] [CLI - PWA mode]
I've tried all major browsers (Chrome, Opera, Samsung, Edge, Brave, etc) and it works fine except Firefox for Android and Firefox for Windows. It does not expand.
This happens using the type='textarea' and multi-line placeholder.

AutoGrow in Chrome for Android:

![1](https://user-images.githubusercontent.com/49736014/210005843-76b12aa4-b91f-4c48-ba4d-87cf2046c5fa.jpg)

AutoGrow in Firefox for Android

![2](https://user-images.githubusercontent.com/49736014/210005875-cfc4e318-a539-450f-a0ac-a1577fde2ee8.jpg)

**Comment by pdanpdan** [comment]
Sorry, there is nothing that can be done for this.
Firefox does not overflow the textarea to fit the placeholder, so there is no way (no simple and efficient way) to detect the placeholder height.

https://stackoverflow.com/questions/34770560/cant-scroll-textarea-placeholder-text-in-firefox

**Reply by MandingaRadi** [reply]
I understand pdanpdan, I'll try to force the height of the div or simply remove the placeholder.
Thank you very much for your answer!
